lib/logstash/codecs/avro.rb

```diff
@@ -71,7 +74,11 @@ def register
 
   public
   def decode(data)
-    datum = StringIO.new(Base64.strict_decode64(data)) rescue StringIO.new(data)
+    if base64_encoding == true
+      datum = StringIO.new(Base64.strict_decode64(data)) rescue StringIO.new(data)
+    else
+      datum = StringIO.new(data)
```

Comment:
@Rubyan maybe we should set `Encoding::ASCII_8BIT` as the encoding of these `StringIO` to avoid portability issues?

Reply:
Yeah that would be even better. Feel free to enhance it.